supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc

```diff
@@ -33,35 +34,27 @@ For documentation questions, contact rhos-docs@redhat.com.
 [discrete]
 [[openstack-services]]
 ==== image:images/yes.png[yes] OpenStack services (noun)
-*Description*: For definitions of all upstream OpenStack services, see the {openstack-glossary}. The following Red Hat OpenStack Platform documentation conventions apply:
-
-* Use the component name instead of the code name.
-** For example: _Block Storage_ instead of _cinder_.
-* If the component name includes _service_, write _service_ in lower case.
-** For example: _Identity service_.
-* Write the code name in lowercase.
-** For example: _nova_.
+*Description*: For definitions of all upstream OpenStack services, see the {openstack-glossary}. The following RHOSP documentation conventions apply:
+
+* Use the component name instead of the code name. For example: _Block Storage_ instead of _cinder_. Exceptions: _heat templates_ (as opposed to _Orchestration service templates_).
+
 * Write the component name in upper case.
```

Comment:
I know this was already here, but would you mind changing this to "uppercase"?

Reply:
Oh yes! missed that. Thanks